underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystemFactory.java

```diff
@@ -101,6 +101,12 @@ private boolean addAndCheckSwiftCredentials() {
       Configuration.set(passwordKeyConf, System.getProperty(passwordKeyConf));
     }
 
+    String swiftSimulationConf = Constants.SWIFT_SIMULATION;
+    if (System.getProperty(swiftSimulationConf) != null
+        && !Configuration.containsKey(swiftSimulationConf)) {
```

Comment:
I decided against a default value in Configuration because it is not applicable except for -PswiftTest test profile. But I did add a default system property.